tests/move_on_sui/basic.move

```diff
@@ -0,0 +1,10 @@
+module xxx {
```

Comment:
this needs to be under tests/parsing/move_on_sui/, it can't be at the top of /tests/